lib/modules/apostrophe-admin-bar/public/js/user.js

```diff
@@ -19,8 +19,14 @@ apos.define('apostrophe-admin-bar', {
 
     self.enhance = function() {
       var $bar = $('[data-apos-admin-bar]');
+      var $isHomepage = $('[data-apos-level="0"]').length;
```

Comment:
Now that this is a simple boolean and not a jquery object anymore... which is great... the `$` is no longer appropriate. Sorry for the hassle. I have a lot of PRs to look at today or I'd slip this change in myself!